packages/web-app-admin-settings/src/index.ts

```diff
@@ -13,13 +13,13 @@ import {
 import { RouteRecordRaw } from 'vue-router'
 
 // just a dummy function to trick gettext tools
-function $gettext(msg) {
+function $gettext(msg: string) {
   return msg
 }
```

Comment:
can't we get rid of this function instead? 

we could do that in a followup as well (and get rid of all of them)...

Reply:
As discussed, let's do this in a follow-up